tests/scikit_learn/etl/test_split.py

```diff
@@ -1,24 +1,178 @@
+import pandas as pd
 from tests.scikit_learn import util
 from juicer.scikit_learn.etl_operation import SplitOperation
 import pytest
 
+
 # Split
-# 
+#
+def test_split_data_integrity():
+    slice_size = 10
+    df = ['df', util.iris(['sepallength', 'sepalwidth',
+                           'petallength', 'petalwidth'], slice_size)]
+
+    arguments = {
+        'parameters': {},
+        'named_inputs': {
+            'input data': df[0],
+        },
+        'named_outputs': {
+            'split 1': 'split_1_task_1',
+            'split 2': 'split_2_task_1'
+        }
+    }
+    instance = SplitOperation(**arguments)
+    result = util.execute(instance.generate_code(),
+                          {'df': df[1]})
+
+    x = [result['split_1_task_1'], result['split_2_task_1']]
```

Comment:
Ok, but you know the outcome. So, you could:
```
assert len(result['split_1_task_1']) == 5
assert len(result['split_2_task_1']) == 5
```

Reply:
i didn't get it...the 'test_split_data_integrity' undo the 'SplitOperation' and then
asserts that the result has the same length and order of dataframe.

Reply:
I didn't get it...the `test_split_data_integrity()` undo the `SplitOperation()` and then asserts that the result has the same length and order as the DataFrame.

Reply:
Ok